lms/djangoapps/course_api/api.py

```diff
@@ -268,3 +269,65 @@ def get_course_run_url(request, course_id):
     """
     course_run_url = reverse('openedx.course_experience.course_home', args=[course_id])
     return request.build_absolute_uri(course_run_url)
+
+
+def get_course_member_queryset(course_key, include_students=True, access_roles=None, prefetch_accessroles=False, prefetch_enrollments=False):
+    """
+    Returns a User queryset that filters all users related to a course.
+    For example - Students, Teachers, Staffs etc.
+
+    Arguments:
+        course_key (CourseKey): the CourseKey for the course
+        include_students: Wether or not to include students,
+        access_roles:
+            accepts an array of string course access roles.
```

Comment:
Please add an example of how to filter out all roles here (e.g `pass an empty dict to exclude all roles (except students)`.

Reply:
Added example calls for different use cases.